src/platform/web/ui/login/PasswordLoginView.js

```diff
@@ -32,15 +32,15 @@ export class PasswordLoginView extends TemplateView {
             disabled
         });
         
-        return t.div({className: "PasswordLoginView form"}, [
+        return t.div({className: "PasswordLoginView form" + (disabled ? " disabled" : "")}, [
```

Comment:
You can do this without string manipulation:
```suggestion
return t.div({
	className: {
		PasswordLoginView: true,
		form: true,
		disabled
	}
}, [
```